src/biogeophys/WaterDiagnosticBulkType.F90

```diff
@@ -277,8 +286,28 @@ subroutine InitBulkHistory(this, bounds)
          fname=this%info%fname('TOTSOILICE'),  &
          units='kg/m2', &
          avgflag='A', &
-         long_name=this%info%lname('vertically summed soil cie (veg landunits only)'), &
+         long_name=this%info%lname('vertically summed soil ice (veg landunits only)'), &
          ptr_col=this%h2osoi_ice_tot_col, l2g_scale_type='veg')
+    ! excess ice vars
```

Comment:
Indentation looks wrong, but maybe was wrong previously?

Reply:
Do you mean the whole block or just the comment? Seems like indents around are the same.

Reply:
Looking at it again, I don't see what I was seeing before. So I'll just resolve this comment.